sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSequencerPartitionContainerDao.java

```diff
@@ -34,7 +37,9 @@
 
   protected static final Logger log = LoggerFactory.getLogger(HibernateSequencerPartitionContainerDao.class);
 
-  private final static String[] SEARCH_PROPERTIES = new String[] { "identificationBarcode" };
+  private static final String[] SEARCH_PROPERTIES = new String[] { "identificationBarcode" };
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "identificationBarcode" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)